tensorboard/uploader/uploader_test.py

```diff
@@ -1127,6 +1196,289 @@ def test_wall_time_precision(self):
         )
 
 
+class TensorBatchedRequestSenderTest(tf.test.TestCase):
```

Comment:
This Test class generously copies from ScalarBatchedRequestSenderTest and focuses on self-contained logic in TensorBatchedRequestSender without the other layers in uploader.py.